### DISCUSSION TITLE
Help! 60k queries for Product Category

### DISCUSSION THREAD

**Discussion by ihearcolors** [PrestaShop/PrestaShop#36499] [Q&A]
![image](https://github.com/PrestaShop/PrestaShop/assets/9917062/726f8006-fd76-4680-a635-58250aa55b5a)
![image](https://github.com/PrestaShop/PrestaShop/assets/9917062/bef799d9-6b26-4ea3-83b3-50d8dbc7e1c4)
This is on default PrestaShop Theme.
PrestaShop Version 8.1.6
VPS, Has enough resources
Longest Query Time: 322ms
![image](https://github.com/PrestaShop/PrestaShop/assets/9917062/f240f147-be7d-45d5-91fe-4f4cce03cdb1)
Queries are fast but way too many, anybody having any idea what is going on?
This happens on default theme on main page, on different theme it happens only on category page, i have a ton of features, tried disabling features in Performance but nothing, uninstalled Faceted Search module, reconfigured it ...whatever, no results.
Everything works pretty fast except loading a category page

![image](https://github.com/PrestaShop/PrestaShop/assets/9917062/83d6d2ba-d816-41e0-bacc-92513c71599d)

Any help or suggestion is welcome!

**Comment by ihearcolors** [comment]
Solved by using 1.7.8.11 instead.

**Reply by kpodemski** [reply]
well, that's not a real solution - there's nothing on PrestaShop 8.1 that could've caused so many extra queries, if you've installed Faceted search module, the question is how many products you have and how many products you've had displayed on 1 page, this sound like something that should've been investigated instead of using version that is no longer fully supported

**Reply by ihearcolors** [reply]
i agree, it should have been investigated but i have to launch the store so it was either try a previous version or change platform, i went with a previous version since i have emagicone's store manager for various tasks.
No. of products were 17k, now with 1.7.8.11 it's 25k and growing, so main suspect were the features...there are lot but it works ok with 1.7.8.11.
Number of products displayed on a page were 20 i think, nothing crazy but to me it did seem like it was querying the whole database.
Faceted search was used, thing is it was worse on default prestashop theme main page than on the custom theme category page.
So it works better on 1.7 with the same modules.
Thanks for your response
